src/types/options/index.ts

```diff
@@ -36,36 +36,62 @@ export type KnownOptionsPayloadFromType<T extends string> = Extract<SlackOptions
  */
 export interface BlockSuggestion extends StringIndexed {
   type: 'block_suggestion';
-  block_id: string;
   action_id: string;
-  value: string;
-
   api_app_id: string;
+  block_id: string;
+  channel?: {
+    id: string;
+    name: string;
+  };
+  is_enterprise_install?: boolean;
+  enterprise?: {
+    id: string;
+    name: string;
+  } | null;
+  message?: {
+    app_id: string;
+    blocks: {
+      block_id: string;
+      type: string;
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      [key: string]: any;
+    }[];
+    is_locked: boolean;
+    latest_reply?: string;
+    metadata?: {
+      event_type: string;
+      event_payload: {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        [key: string]: any;
+      };
+    };
+    reply_count: number;
+    reply_users: string[];
+    reply_users_count: number;
+    team: string;
+    text: string;
+    thread_ts: string;
+    ts: string;
+    type: 'message';
+    user: string;
+  };
   team: {
     id: string;
     domain: string;
     enterprise_id?: string;
     enterprise_name?: string;
   } | null;
-  channel?: {
-    id: string;
-    name: string;
-  };
   user: {
     id: string;
     name: string;
     team_id?: string;
   };
-  token: string; // legacy verification token
```

Comment:
While next-gen function interactivity event payload won't contain this field, it may still be being sent for all other block_suggestions events and/or relied upon by users. This is one of those fields deno-slack-sdk block_suggestions schema can omit but we probably shouldn't omit just yet.

Reply:
I do see you have the catch-all field tho..assuming that's from the other schema though. I feel like the deno_slack_sdk version isn't so different from our internal one and _could_ technically be something we consume from a shared types. We can probably note that and move on for the time being though.

Reply:
sounds good! I can add the `token` field back in since it was in there already and also to be more explicit about it being expected in this type of payload for some scenarios. thoughts on adding it in and making it `optional` since it's for legacy only?

Reply:
Yeup, making it optional sounds good to me!